web/tendrl/multiuser_cluster_import.rst

```diff
@@ -1,2 +1,52 @@
-multiuser_cluster_import
-*******************************************************************************
\ No newline at end of file
+Multiuser - Cluster import
+****************************
+
+:authors:
+          - mkudlej@redhat.com
+
+
+Description
+============
+
+Import cluster simultaneously from two browsers.
+
+Setup
+======
+
+#. Prepare environment as described in :doc:`/configurations/multiuser_initial_setup`.
```

Comment:
Do you mean that you should log in with 2 different admin accounts, each in a separate browser?

Reply:
Would this test case be valid or the same if you log in with the same admin account from 2 different browsers?